CONTRIBUTING.md

```diff
@@ -1,21 +1,22 @@
+# Contributing
+
 # Issues
-Before opening an issue, please make sure your problem/request doesn't exist. When you open issue, please provide the following information if possible:
+Before opening an issue, please make sure your problem or request doesn't exist. When opening an issue, please provide the following information if possible:
 
 * Example code or repo (please keep it simple and minimal)
-* Steps to reproduce
-* `react-bootstrap-table2` version
+* Steps to reproduce.
+* `react-bootstrap-table2` version.
 
-Anyway, you're welcome to open an issue to ask questions or discuss about a feature request.
+Anyway, you're always welcome to open an issue to ask questions or talk about new feature request.
 
 # Pull Requests
 Check [here](./docs/development.md) for getting started with development.
 
-* When you want to implement a new feature, please let us know (via issues).
-* Please run tests before opening a PR and also remember to test the actual business logic.
-* If your PR is trying to fix a bug, please describe the bug number with hashtag when creating the PR.
+* We recommend filing an [issue](https://github.com/react-bootstrap-table/react-bootstrap-table2/issues) before you implement any new features.
+* Please ensure all the test suit passes before submitting a PR. And always test the actual business logic.
+* If your PR is for fixing bug, please describe the bug when submitting the PR. and if possible, including the bug number with hastag.
 
 # For the members of react-bootstrap-table2 org
-
-* Please convert the ticket to issue when is ticket moved from `Backlog` to `Ready` in project.
-* Please write documentation if any new API/feature/props is changed or added.
-* Please add a story example if any new feature is implemented
\ No newline at end of file
+* Please convert the ticket to issue when the ticket has moved from `Backlog` to `Ready`.
+* Please update the docs with any API, feature, component props changes or added, the code and docs should always be in sync.
+* Please add a story example if any new feature is implemented.
```

Comment:
Please add a story example if any new feature was implemented.

Reply:
Ah, good catch! Yes, it should be "was" not "is".